.travis.yml

```diff
@@ -19,17 +16,15 @@ branches:
 matrix:
     fast_finish: true
     include:
-        - php: 7.1
-          env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" DEPENDENCIES="doctrine/instantiator:^1.1"
+        - php: 7.0
```

Comment:
This was previously on PHP 7.1 due to doctrine/istantiator 1.1, which is no longer required. I've added a conflict rule below.